source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md

```diff
@@ -16,23 +16,23 @@ Enable SSL before updating DNS. SSL for custom domains is available for Professi
 
 ## Steps to Enable SSL
 
-1. Generate Key and CSR
+1. Generate RSA Key and CSR
 2. Get SSL certificate
 3. Load certificate into Pantheon environment
 4. Test SSL (optional, but recommended)
 5. Update DNS
 6. Require SSL for all pages (optional, but recommended)
 
-## Generate Key and CSR
+## Generate RSA Key and CSR
 
-Run `openssl` from the command line to generate a RSA Private Key (.key file) and Certificate Signing Request (.csr file):
+Run `openssl` from the command line to generate an [RSA private key](https://en.wikipedia.org/wiki/RSA_(cryptosystem)) (.key file) and [certificate signing request](https://en.wikipedia.org/wiki/Certificate_signing_request) (.csr) file:
 
 ```bash
 openssl req -new -newkey rsa:2048 -nodes -out www_example_com.csr
 -keyout www_example_com.key
 ```
 
-You'll be prompted interactively to enter information needed to request your certificate. The most important part of this information is the *Common Name*, which is the domain.
+You'll be prompted interactively to enter information the needed to request your certificate. The most important part of this information is the *Common Name*, which is the domain.
```

Comment:
Remove "the" in "You'll be prompted interactively to enter information the needed to request your certificate."